src/schemathesis/cli/junitxml.py

```diff
@@ -38,5 +36,5 @@ def handle_event(self, context: ExecutionContext, event: events.ExecutionEvent)
                     message=event.result.errors[-1].exception, output=event.result.errors[-1].exception_with_traceback
                 )
         if isinstance(event, events.Finished):
-            ts = [TestSuite("schemathesis", test_cases=self.test_cases, hostname=platform.node())]
-            to_xml_report_file(file_descriptor=self.file_handle, test_suites=ts, prettyprint=True)
+            testsuites = [TestSuite("schemathesis", test_cases=self.test_cases, hostname=platform.node())]
```

Comment:
"testsuites" -> "test_suites" 
So the variable has the same name as the keyword argument in `to_xml_report_file`